lib/test/tasks/run_file_size_checks.rb

```diff
@@ -17,7 +17,7 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
     'groceries*.css' => (108..118),
     'groceries*.js' => (441..451),
     'home*.css' => (0..10),
-    'home*.js' => (239..249),
+    'home*.js' => (260..270),
```

Comment:
It's sort of sad to be increasing the page weight by ~20 KiB with this change, but I am pretty interested in tracking these scroll events, so I think it's worth it.